client/app/cloud/project/details/translations/Messages_fr_FR.xml

```diff
@@ -2,13 +2,13 @@
 <translations>
    <translation id="cpd_project_redirect" qtlid="245857">Redirection...</translation>
 
-   <translation id="cpd_project_suspended" qtlid="263858">Votre projet cloud est suspendu. Pour plus d'informations, merci de contacter <a href="#/support">le support</a>.</translation>
+   <translation id="cpd_project_suspended">Votre projet cloud est suspendu. Pour plus d'informations, merci de contacter <a href="{{ url }}">le support</a>.</translation>
```

Comment:
No `target="_blank"`, `rel="noopener"` ?

Reply:
Nope because it's to stay in manager environment